sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -239,11 +239,7 @@ Object groundingHelpSapCom(
   @Nonnull
   Object responseFormatJsonSchema(
       @RequestParam(value = "format", required = false) final String format) {
-    final var response = service.responseFormatJsonSchema("apple");
-    if ("json".equals(format)) {
-      return response;
-    }
-    return response.getContent();
+    return service.responseFormatJsonSchema("apple");
```

Comment:
(Question)

What is this change?

Reply:
The return type is class Translation, it can't be json or text

Reply:
Why do we still have the `format` argument on the mapping above when it's ignored?
Also if the return type is definite, you can change it from `Object` to `Translation`.

*However* I would like our demo app to stay consistent, either all endpoints support `text` + `object` format or all endpoints support only `object` format.
[Here's what it would look like with format param re-established](https://github.com/SAP/ai-sdk-java/compare/entity...entity2). I know it's bad. But I would rather have this than inconsistency. 

Alternatively remove the `text` format globally 🤷 would be okay with me too.